#Global parameters
offline: false
fixed: true
draw: true

publish_point_cloud: false

#Arm topics
joint_state_topic: "/arm5e/joint_state_angle"
joint_state_command_topic: "/arm5e/command_angle"

#Joint values for initial and final scanning configurations
scan_initial_posture: [0.0, 0.6, 0.7, 0.0, 0.3]
scan_final_posture: [ 0.0, 0.6, 1.5, 0.0, 0.1]

#Names of cameras
#cameras: ["stereo_down/left", "eye", "stereo_down/right", "stereo_real/left"]
cameras: [ "stereo_real/left"]

#stereo_down/left
stereo_down/left/image: "stereo_down/left/image_raw"
stereo_down/left/camera_info: "stereo_down/left/camera_info"
stereo_down/left/reconstruction_type: "static"
stereo_down/left/bMc: [0.240, -0.177, 0.242, 0.001, 0.001, -0.707, 0.707]
stereo_down/left/eMl: [-0.092, -0.00, -0.188, 0.0, 0.0, 0.0, 1.0]
stereo_down/left/peak_detector_type: "simulation"
stereo_down/left/min_radius_WS: 0.5
stereo_down/left/max_radius_WS: 2.0
stereo_down/left/base_name: "/home/toni/3DReconstructions/stereo_left_thread"


#eye
eye/image: "uwsim/eye"
eye/camera_info: "uwsim/eye_info"
eye/reconstruction_type: "eye"
eye/eMc: [0.041, 0.177, -0.163, -0.141, -0.141, 0.693, 0.693]
eye/eMl: [-0.092, -0.000, -0.188, 0.0, 0.0, 0.0, 1.0]
eye/peak_detector_type: "simulation"
eye/min_radius_WS: 0.2
eye/max_radius_WS: 2.0
eye/base_name: "/home/toni/3DReconstructions/eye"


#stereo_down/right
stereo_down/right/image: "stereo_down/right/image_raw"
stereo_down/right/camera_info: "stereo_down/right/camera_info"
stereo_down/right/reconstruction_type: "static"
stereo_down/right/bMc: [-0.5, -0.06, -0.13, 0.001, 0.001, -0.707, 0.707]
stereo_down/right/eMl: [-0.092, 0.0, -0.188, 0.0, 0.0, 0.0, 1.0]
stereo_down/right/peak_detector_type: "simulation"
stereo_down/right/min_radius_WS: 0.5
stereo_down/right/max_radius_WS: 2.0
stereo_down/right/base_name: "/home/toni/3DReconstructions/stereo_right_thread"


#stereo_real/left
stereo_real/left/image: "/stereo/left/image_raw"
stereo_real/left/camera_info: "/stereo/left/camera_info"
stereo_real/left/reconstruction_type: "static"
stereo_real/left/bMc: [-0.32347, 0.045,  0.07675, 0.0, 0.0, -1.57]
stereo_real/left/eMl: [-0.02, 0.0, 0.12, 0.0, 0.0, 0.0]
stereo_real/left/peak_detector_type: "last_image"
stereo_real/left/min_radius_WS: 0.0
stereo_real/left/max_radius_WS: 0.0
stereo_real/left/base_name: "/home/toni/3DReconstructions/stereo_left_real_new"

 


